### ISSUE TITLE
iOS - onPartialResult null string value

### ISSUE THREAD

**Issue by SuavePirate** [riderodd/react-native-vosk#47]
Hello! We have this implemented for both Android and iOS - Android works like a charm, but on iOS we get a null value passed in for the `onPartialResult` callback function.

Example logs output from both the `onPartialResult` and the `onResult` on iOS:

```
 LOG  partial result:  null
 LOG  partial result:  null
 LOG  partial result:  null
 LOG  partial result:  null
 LOG  partial result:  null
 LOG  final result:  check check one to one two
 ```
 
 So we can see that the event is firing but no value for `res` in 
 ``` js
const partialResultEvent = vosk.onPartialResult(async (res) => {
            console.log('partial result: ', res);
});
const resultEvent = vosk.onResult(async (res: string) => {
            console.log('final result: ', res);
});
 ```
 
 Any guidance or workaround would be greatly appreciated!

**Comment by joncamp** [comment]
What are our thoughts on the timing of this - should we give them till the weekend to update the release, or do we want to temporarily check-in a locally built library until then?

**Comment by riderodd** [comment]
I'll Check on that wednesday, sorry for the delay 😅

Le lun. 25 mars 2024, 23:04, Jon Campbell ***@***.***> a
écrit :

**Comment by SuavePirate** [comment]
Thanks @riderodd! Btw it looks like #48 fixes it - tested on my devices and partials are coming through

**Comment by riderodd** [comment]
Fixed on 0.2.2
